_plugins/statistics_tag.rb

```diff
@@ -16,6 +23,16 @@ def self.get_stats(site)
         'states' => {}
     }
 
+    unless institutes.has_key?(people['institutes'])
+      raise StatisticsError.new 'Institute not found. Typo? Look at data/institutes.yml!'
+    end
+
+    unless people['position'].is_nil?
```

Comment:
... neither this.

We should check below in `people.each`, whether `institutes.has_key? person['affiliation']` and `positions.has_key? person['position']`. That's where actual user input is used as keys of pre-constructed maps.